src/main/webapp/app/exam/participate/exercises/modeling/modeling-exam-submission.component.ts

```diff
@@ -26,6 +26,10 @@ export class ModelingExamSubmissionComponent extends ExamSubmissionComponent imp
     exercise: ModelingExercise;
     umlModel: UMLModel; // input model for Apollon
 
+    constructor(private cdr: ChangeDetectorRef) {
```

Comment:
please use a proper name and not `cdr`

Reply:
See 5e32a09205c4f75d49ba9065afcb6e3699d48857